src/core/cdk/src/initial-setup.ts

```diff
@@ -905,7 +905,7 @@ export namespace InitialSetup {
         .next(addTagsToSharedResourcesTask)
         .next(enableDirectorySharingTask)
         .next(deployPhase5Task)
-        .next(createAdConnectorTask)
+        // .next(createAdConnectorTask)
```

Comment:
Hey,
The prescribed way to disable the deployment of the AD Connector (ADC) is to simply not include it in any ```deployments``` section of the config.json file, or if included set the value of ```deployments\adc\deploy``` to ```false```.